plugins/plugin-tutorials/web/html/index.html

```diff
@@ -34,26 +36,30 @@ <h2 class="tutorial-content-extras-title">Learn More</h2>
 
               <div class="tutorial-code-snippet bx--snippet bx--snippet--multi tutorial-hide-when-minimized">
                 <div class="bx--snippet-container" aria-label="Code Snippet">
-                  <pre><code></pre></code>
+                  <pre>
+                    <code></code>
+                  </pre>
                 </div>
-              </div> <!-- code snippet -->
-
-            </div> <!-- tutorial-left -->
+              </div>
+              <!-- code snippet -->
+            </div>
+            <!-- tutorial-left -->
 
             <div class="tutorial-right">
-              <div class="tutorial-content-extras tutorial-hide-when-minimized scrollable scrollable-auto" tabIndex="0">
-                <div class="tutorial-content-extras-as-structured-list tutorial-template"
-                     id="tutorial-structured-list-template">
+              <div class="tutorial-content-extras tutorial-hide-when-minimized scrollable scrollable-auto" tabindex="0">
+                <div class="tutorial-content-extras-as-structured-list tutorial-template" id="tutorial-structured-list-template">
                   <h2 class="tutorial-content-extras-title">Next Steps</h2>
                   <div class="tutorial-content-extras-body bx--structured-list">
                     <div class="bx--structured-list-thead">
                       <div class="bx--structured-list-row bx--structured-list-row--header-row">
                         <!-- header row -->
                       </div>
-                    </div> <!-- structured list thead -->
-                    <div class="bx--structured-list-tbody">
-                    </div> <!-- structured list tbody -->
-                  </div> <!-- structured list -->
+                    </div>
+                    <!-- structured list thead -->
```

Comment:
the point of these comments was to help understand which element was being closed. it seems like, after moving the comment to the next line, the comments need comments....

Reply:
Ouch! We can't customize it. Do you prefer to remove Prettier from HTML files?

Reply:
i think we either need to remove these "closing comments" or not move them to a new line. with this PR as it stands, the closing comments become quite confusing

Reply:
We can not avoid moving it, Prettier does not have that configuration.

Do you prefer to remove comments or remove Prettier from HTML files?

Reply:
let's remove those closing comments, thanks!

Reply:
Updated!